docs/pages/server-access/guides/tsh.mdx

```diff
@@ -482,3 +483,19 @@ This is possible even if nodes in the `production` cluster are located behind a
 firewall without open ports. This works because the `production` cluster
 establishes a reverse SSH tunnel back into `work` proxy, and this tunnel is used
 to establish inbound SSH connections.
+
+## Configuration file
+
+`tsh` has an optional configuration file that can be created in `$HOME/.tsh/config/config.yml`.
+This file allows specifying HTTP headers to include when making requests 
+to proxies matched on the `proxy` field.
```

Comment:
Are these Proxy Service instances? If so, we should say "Teleport Proxy Service instances" (or maybe "Teleport Proxy Servers") instead of "proxies."

If not, we should clarify.